README.md

```diff
@@ -7,12 +7,12 @@ The Endpoint Querier takes one arguement, a JSON file containing the endpoints w
 
 ```bash
 go get ./...
-go install ./...
+go mod download
```

Comment:
running `go get ./...` gave me this output:

```
go get: warning: modules disabled by GO111MODULE=auto in GOPATH/src;
	ignoring go.mod;
	see 'go help modules'
```

which doesn't seem right. is this command no longer needed?